contracts/src/bridge/L1/TenBridge.sol

```diff
@@ -132,10 +132,12 @@ contract TenBridge is
         address receiver
     ) private {
         SafeERC20.safeTransfer(IERC20(asset), receiver, amount);
+        emit Withdrawal(receiver, asset, amount);
     }
 
-    function _receiveNative(address receiver) private {
-        (bool sent, ) = receiver.call("");
+    function _receiveNative(address receiver, uint256 amount) private {
+        (bool sent, ) = receiver.call{value: amount}("");
         require(sent, "Failed to send Ether");
+        emit Withdrawal(receiver, address(0), amount);
```

Comment:
shouldn't there be another security check here?

This bit:
```Ethereum -> Ethereum: Check that the message originated from the `"EthereumBridge"` (LogMessagePublished.sender=address(EthereumBridge))```
( https://github.com/ten-protocol/go-ten/pull/2406/files#diff-8cb08d686a43a7cc267639355d881e8ff7a66b9fb92df64a31daa1e503eeedadR93 )

Reply:
onlyCrossChainSender(remoteBridgeAddress) does this